t/27-consoles-sshXtermVt.t

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/perl
+
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+use Mojo::Base -strict, -signatures;
+use Test::Most;
+use Test::Warnings qw(:report_warnings);
+use Test::MockObject;
+use Test::MockModule;
+use Test::Output qw(stderr_from);
+use tinycv;
```

Comment:
that should better be moved into VNC.pm, like I wrote yesterday. 
VNC.pm is using a function from tinyvc, so it should use the module itself

Reply:
I just found that the dependency isnt needed anymore. I guess because the mocks which was added afterward. So I removed it. Do you think still need to add it to VNC.pm?

Reply:
I think the `use tinyvc` belongs there, but since this PR is not touching it anymore directly or indirectly, this should be done in its own PR